test/middleware/csp.test.js

```diff
@@ -177,7 +177,7 @@ describe('CSP middleware', () => {
     const csp = requireHelper('../../app/middleware/csp')
 
     const next = sinon.spy()
-    const response = { setHeader: sinon.spy() }
+    const response = { setHeader: sinon.spy(), removeHeader: sinon.spy() }
```

Comment:
Just wondering why we need to add a spy for this function?

Reply:
the helmet upgrade, presume it now removes headers in some cases and this was required to make the tests green